custom-tests.json

```diff
@@ -205,232 +205,232 @@
       "__base": "if (!reusableInstances.audioContext) {return false;} var instance = reusableInstances.audioContext.createGain();"
     },
     "HTMLAnchorElement": {
-      "__base": "var instance = document.createElement('a'); if (instance.constructor.name === 'HTMLUnknownElement' || instance.constructor.name === 'HTMLElement') {return false;}"
+      "__base": "var instance = document.createElement('a'); if (instance.constructor.name =!= 'HTMLAnchorElement') {return false;}"
```

Comment:
This diff is delightful, I gotta suspect an `!` was inserted at the same place in many lines, and it didn't quite work out :D

Reply:
It was, something odd happened with the multiple selection on my editor.  Fixed!